data/persons.txt

```diff
@@ -1885,30 +1880,37 @@ person "Patrol Team"
 			"		How do we indent after line breaks again?"
 			"Please put the includes in alphabetical order."
 			"Always keep three empty lines between functions."
-			"Why do Aliens speak English?"
-
-#		1010todd
-			"Style check."
-			"I'll give this a better look later."
-			"What is this for?"
-			"This should be a private member."
-			"Comments?"
-			"Please remove unnecessary space."
-
-			"Looks good to me."
-			"Is this the best name?"
-			"The standard copyright header was changed recently."
-
-			"Please use the template."
-			"The decision to implement fancy quote is one of the worst in computer history."
-			"	This needs another indent."
+			"Why do aliens speak English?"
+
+#		bene-dictator
```

Comment:
Why are these comments at the beginning with each line, and indented after the `#`, instead of the other way around?

Reply:
Idk, I saw Saugia do it, then just decided to do it his way.

Reply:
Hashes generally look more appropriate if they are at the beginning of a line, not immediately before a phrase. It's more personal preference though, other files do it both ways.